spec/unit/provider/oneview_logical_interconnect_spec.rb

```diff
@@ -16,10 +16,9 @@
 
 require 'spec_helper'
 require_relative '../../support/fake_response'
-require_relative '../../shared_context'
```

Comment:
why did u removed this..?

Reply:
this is not requited as we have include_contet for this purpose

Reply:
since spec_helper is already inheriting shared_context, its not required to inherited again.